blamo/lib/main.dart

```diff
@@ -39,6 +57,9 @@ class HomePage extends StatelessWidget {
   }
 }
 
+/* This is where Users will be creating and filling out documents for logging
+* This second page will house the text fields and (presumably) imaging options
+*/
 class SecondPage extends StatelessWidget {
```

Comment:
Depending extent of functionality of this page we should slap it into a new .dart file and import in otherwise we gonna get messy

Reply:
This portion was mostly to demonstrate the use of the floating button. We can wire it up to open a new page and just remove the blank page.